src/components/layout/WithVehicleReference.stories.mdx

```diff
@@ -47,28 +52,25 @@ export const Template = (args) => <LayoutWithVehicleReference {...args} />;
 
 # Layout with vehicle reference
 
-## With back link
+## With title
 
 <Canvas>
   <Story
-    name="With back link"
+    name="With title"
     args={{
-      backLink: {
-        text: 'Back',
-        url: 'https://www.autoscout24.ch/de',
-      },
+      title: 'Nachricht an den Verkäufer',
     }}
   >
     {Template.bind({})}
   </Story>
 </Canvas>
 
-<ArgsTable story="With back link" />
+<ArgsTable story="With title" />
```

Comment:
could we please keep the "with back link" version?

Reply:
because I think it is broken if you have no title and no backlink because of the minmax property that is wrong in that scenario.
![image](https://user-images.githubusercontent.com/71456764/185394862-e9db8ad1-4849-43a6-9ff0-100551313da1.png)

Reply:
sure thing! 🐥